src/runtime/internal/style_manager.ts

```diff
@@ -25,14 +27,18 @@ export function create_rule(node: Element & ElementCSSInlineStyle, a: number, b:
 
 	const rule = keyframes + `100% {${fn(b, 1 - b)}}\n}`;
 	const name = `__svelte_${hash(rule)}_${uid}`;
+	const doc = node.ownerDocument;
+	activeDocs.add(doc);
+	let [ stylesheet, current_rules ] = (docStyles.get(doc) || []) as DocStyles;
+	if (!stylesheet) {
+		current_rules = {};
+		const style = element('style');
+		doc.head.appendChild(style);
+		stylesheet = style.sheet as CSSStyleSheet;
+		docStyles.set(doc, [ stylesheet, current_rules ]);
+	}
```

Comment:
This is obnoxious code-golfing, but if you wanted to save 18 bytes (after minification) you could do this...

```typescript
const doc = node.ownerDocument;
activeDocs.add(doc);
const [stylesheet, current_rules] = (docStyles.has(doc) ? docStyles : docStyles.set(doc, [
	(doc.head.appendChild(element('style') as HTMLStyleElement).sheet as CSSStyleSheet),
	{}
])).get(doc) as DocStyles;
```

Can we change `docStyles` to `doc_styles`?